README.md

```diff
@@ -1,2 +1,60 @@
 # sumologic-kubernetes-collection-helm-operator
-Helm Operator for the Sumo Logic Kubernetes Collection Helm Chart
+
+Sumo Logic Kubernetes Collection Helm Operator for the [Sumo Logic Kubernetes Collection Helm Chart][helm-chart-repo].
+
+**Supported platforms:**
+
+- OpenShift 4.6
+- OpenShift 4.7
+
+## Installation
+
+### From OperatorHub
+
+- Install Sumo Logic Kubernetes Collection Helm Operator from Operator Hub
```

Comment:
Can we link to the Operator Hub?

Reply:
```suggestion
- Install Sumo Logic Kubernetes Collection Helm Operator from the [Operator Hub](https://operatorhub.io/)
```

Reply:
Helm operator is not published now and I'm not sure if when it will be published it will be available from this website or only from "Operator Hub" tab in OpenShift web console... I prefer to update this later